components/Images/Images.css

```diff
@@ -0,0 +1,15 @@
+
+.img_responsive{
+     width:50rem;
+     height:30rem;
+}
+.img_round{
+        width: 13rem;
+        height:13rem;
+        border-radius: 50%;
+    
+}
+.img_square{
+    width:14rem;
+    height:14rem;
```

Comment:
I think giving an image fixed width will be problematic from a reusability point of view. What you can do is create a `div` around the image and then give it sizes.

Reply:
Okay